src/BloomExe/Book/Book.cs

```diff
@@ -1362,6 +1362,21 @@ private IPage GetPageToShowAfterDeletion(IPage page)
 			return _pagesCache[index + 1]; //give the following page
 		}
 
+		public Dictionary<string, IPage> GetTemplatePagesIdDictionary()
```

Comment:
Made some safety changes. It shouldn't get called if it's not a template (since the button won't be enabled; so far at least).